legacy/Makefile.include

```diff
@@ -84,6 +86,8 @@ LDFLAGS  += -L$(TOP_DIR) \
             $(CPUFLAGS) \
             $(FPUFLAGS)
 
+ZKP_PATH = ../../vendor/secp256k1-zkp
```

Comment:
Although this works, to be consistent I think we should respect the current convention of referencing 3rd party code through the `legacy/vendor` directory, which should have a symlink to the root vendor directory.

Reply:
Good point, fixed in e981bc7f1bed31e9f349aaf1b77c803a1f474f20.